draft-ietf-taps-arch.md

```diff
@@ -484,13 +473,13 @@ Clients need to ensure that security APIs are used appropriately. In cases where
 clients use an interface to provide sensitive keying material, e.g., access
 to private keys or copies of pre-shared keys (PSKs), key use needs to be
 validated. For example, clients ought not to use PSK material created for the
-Encapsulating Security Protocol (ESP, part of IPsec) {{RFC4303}} with QUIC, and clients
+Encapsulating Security Protocol (ESP, part of IPsec) {{?RFC4303}} with QUIC, and clients
 ought not to use private keys intended for server authentication as a keys for
 client authentication.
 
 Moreover, unlike certain transport features such as TCP
-Fast Open (TFO) {{RFC7413}} or Explicit Congestion Notification (ECN)
-{{RFC3168}} which can fall back to standard configurations, Transport
+Fast Open (TFO) {{?RFC7413}} or Explicit Congestion Notification (ECN)
```

Comment:
Another new request, sorry again to only bring this up now: the mention of TFO and ECN here seems strange, because these mechanisms are internal to a protocol (the TCP implementation can try and fall back), whereas our transport system would just use protocols as they are. For that reason, these things do also not match our definition of "Transport Features" (as defined in RFC 8095, and then repeated in RFC 8303 and minset). I would just remove all the text from "unlike ...."  until "... standard configurations".

Reply:
Fixed!